modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/fork/handler.scala

```diff
@@ -0,0 +1,25 @@
+package org.tessellation.sdk.infrastructure.fork
+
+import cats.effect.Async
+import cats.syntax.flatMap._
+import cats.syntax.show._
+
+import org.tessellation.schema.gossip.PeerRumor
+import org.tessellation.sdk.domain.fork.{ForkInfo, ForkInfoStorage}
+import org.tessellation.sdk.infrastructure.gossip.{IgnoreSelfOrigin, RumorHandler, rumorLoggerName}
+
+import org.typelevel.log4cats.slf4j.Slf4jLogger
+
+object handler {
+
+  def forkInfoHandler[F[_]: Async](storage: ForkInfoStorage[F]): RumorHandler[F] = {
```

Comment:
Would this be more clear if the object was called `ForkInfoRumorHandler` and the function called `make`?

Reply:
I am following the convention in the other `handler` files.

Reply:
There are also examples of a well-named `object` and `make` or other descriptive name. I think that's a better pattern to follow.